packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -2,6 +2,28 @@
 title: Okta API Products Release Notes
 ---
 
+## April
+
+### Monthly Release 2021.04.0
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.5.0](#okta-sign-in-widget-version-550) | March 31, 2021 |
+| [Custom Domain API now in EA in Preview](#custom-domain-api-is-now-in-ea-in-preview) | March 31, 2021 |
+| [Groups API extended search now GA in Production](#groups-api-extended-search-is-now-ga-in-production) | March 31, 2021 |
+
+#### Okta Sign-In Widget, version 5.5.0
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.5.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/).
+
+#### Custom Domain API is now in EA in Preview
+
+The [Custom Domain API](/docs/reference/api/domains/) is now in Early Access (EA) in Preview.
```

Comment:
Assume this API will be merged into the Release branch later (link throwing a 404)

Reply:
Yes - it's part of the 2021.04.0 release branch